docs/sources/dashboards/share-dashboards-panels/_index.md

```diff
@@ -1,18 +1,14 @@
 ---
 aliases:
-  - ../administration/reports/
-  - ../enterprise/export-pdf/
-  - ../enterprise/reporting/
-  - ../reference/share_dashboard/
-  - ../reference/share_panel/
-  - ../share-dashboards-panels/
-  - ../sharing/
-  - ../sharing/playlists/
-  - ../sharing/share-dashboard/
-  - ../sharing/share-panel/
+  - ../reference/share_dashboard/ # /docs/grafana/latest/reference/share_dashboard/
+  - ../reference/share_panel/ # /docs/grafana/latest/reference/share_panel/
+  - ../share-dashboards-panels/ # /docs/grafana/latest/share-dashboards-panels/
+  - ../sharing/ # /docs/grafana/latest/sharing/
+  - ../sharing/playlists/ # /docs/grafana/latest/sharing/playlists/
+  - ../sharing/share-dashboard/ # /docs/grafana/latest/sharing/share-dashboard/
+  - ../sharing/share-panel/ # /docs/grafana/latest/sharing/share-panel/
   - ./
```

Comment:
@jdbaldry - I can't figure out what the description is here

Reply:
I believe that it will be:

```suggestion
  - ./ # /docs/grafana/latest/dashboards/
```

Reply:
I can't confidently say that was intended though looking at the original PR that changed from absolute to relative aliases: https://github.com/grafana/grafana/commit/c1c48dd610ec3a4a02469daf22e56b402b9e731e#diff-c39813d71dfd771cfdf40cb9c23319bf839203f3bced7682304fe6f453e455ae.

Probably safe to remove since the page is the parent topic.

```suggestion
```